ddtrace/contrib/aws_lambda/__init__.py

```diff
@@ -28,9 +28,12 @@
 
 .. py:data:: DD_APM_FLUSH_DEADLINE_MILLISECONDS
 
-   When to flush unfinished spans in an impending timeout.
+   Time in milliseconds used to define the flush deadline for traces.
+   Example: if the remaining time in an AWS Lambda is 1000 milliseconds,
+   and the environment variable is set to 200 we would be flushing at 800 
+   milliseconds.
```

Comment:
The description is a bit confusing. Here's a suggestion but feel free to edit/ignore it: 
```suggestion
   Used to determine when to submit spans in aws lambda functions before a timeout occurs. When the time remaining in a lambda function is less than `DD_APM_FLUSH_DEADLINE_MILLISECONDS` the tracer will attempt to submit the current active span and all finished spans.
```

Reply:
Tweaking it a little bit. After re-reading it many times, I settled up with 
> Used to determine when to submit spans before a timeout occurs. When the time remaining in an AWS Lambda function is less than `DD_APM_FLUSH_DEADLINE_MILLISECONDS`, the tracer will attempt to submit the current active spans and all finished spans.